.github/workflows/test-conda-build.yml

```diff
@@ -30,4 +30,6 @@ jobs:
         shell: bash -l {0}
         run: |
           conda install datacube
-          datacube --help
+          pip install -e .
```

Comment:
preferred way these days is `python -m pip` instead of just `pip`... something to do with PATH.